src/components/schemas/fields.yaml

```diff
@@ -192,6 +222,63 @@ source:
     Enum value signifying the client from which this expense was created
   example: SLACK
 
+expense_policy_source:
+  type: string
+  enum:
+    - WEBAPP_POLICY_FORM
+    - WEBAPP_POLICY_JSON
+    - WEBAPP_EXPENSE_RECEIPT_AMOUNT_LIMIT
+    - WEBAPP_EXPENSE_MILEAGE_SLAB_RATE
+  description: >
+    Value signifying the client from which this expense policy is created
+  example: WEBAPP_POLICY_FORM
+
+expense_policy_limit_type:
+  type: string
+  enum:
+    - INDIVIDUAL
+    - DAILY
+    - NIGHTLY
+    - WEEKLY
+    - MONTHLY
+    - QUARTERLY
+    - HALF_YEARLY
+    - YEARLY
+  description: >
+    Value signifying the limit type of the expense policy. <br><br>
+    `INDIVIDUAL`: Amount limit will be checked against each expense. <br>
+    `DAILY`: Amount limit will be checked against the total of all expenses in a day. <br>
+    `NIGHTLY`: Amount limit will be checked against the hotel cost per night. <br>
+    `WEEKLY`: Amount limit will be checked against the total of all expenses in a week. <br>
+    `MONTHLY`: Amount limit will be checked against the total of all expenses in a month. <br>
+    `QUARTERLY`: Amount limit will be checked against the total of all expenses in a financial quarter. <br>_Note: expense_fiscal_year_start_month is required when you set this._ <br>
```

Comment:
- Note: field `expense_fiscal_year_start_month` is required if you set `QUARTERLY` for this field.

- Note: field `expense_fiscal_year_start_month` is required if you set `QUARTERLY` for `expense_policy_limit_type` field